hatchet_sdk/clients/admin.py

```diff
@@ -262,30 +257,22 @@ async def run_workflow(
     async def run_workflows(
         self,
         workflows: list[WorkflowRunDict],
-        options: TriggerWorkflowOptions | None = None,
-    ) -> List[WorkflowRunRef]:
+        options: TriggerWorkflowOptions = TriggerWorkflowOptions(),
+    ) -> list[WorkflowRunRef]:
         if len(workflows) == 0:
             raise ValueError("No workflows to run")
         try:
             if not self.pooled_workflow_listener:
                 self.pooled_workflow_listener = PooledWorkflowRunListener(self.config)
 
-            namespace = self.namespace
-
-            if (
-                options is not None
-                and "namespace" in options
-                and options["namespace"] is not None
-            ):
-                namespace = options["namespace"]
-                del options["namespace"]
+            namespace = options.namespace or self.namespace
```

Comment:
There are lots of lines like this throughout the codebase that are vastly simplified with Pydantic / dataclasses / etc. with better defaults since we don't need to check for `None` and for the existence of a key before trying to access that key from `options` (e.g.)